app/src/main/java/com/github/ashutoshgngwr/noice/Constants.kt

```diff
@@ -0,0 +1,5 @@
+package com.github.ashutoshgngwr.noice
+
+object Constants {
+  const val SELECTED_PRESET_ID = "selectedPresetID"
```

Comment:
Can we move this constant to WakeUpTimerFragment's companion? A separate object to declare constants seems unnecessary since only a single class is using the constant in question.

Reply:
Of course, that solution is much better.